src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -129,6 +131,21 @@ public ExtractCohortEngine(final String projectID,
         this.refSource = refSource;
         this.sampleIdToName = sampleIdToName;
         this.sampleNames = new HashSet<>(sampleIdToName.values());
+        this.sampleIdsToExtract = new TreeSet<>(this.sampleIdToName.keySet());
+
+        long maxSampleId = sampleIdsToExtract.last();
+
+        // Constraint due to easily available BitSet implementations, could
+        // be extended in future work
+        if (maxSampleId > Integer.MAX_VALUE) {
+            throw new GATKException("Sample Ids > " + Integer.MAX_VALUE + " are not supported");
+        }
+
+        this.sampleIdsToExtractBitSet = new BitSet(sampleIdsToExtract.last().intValue());
```

Comment:
Since you already have it in a local?
```suggestion
        this.sampleIdsToExtractBitSet = new BitSet(maxSampleId.intValue());
```

Reply:
👍